lasagne/layers/special.py

```diff
@@ -633,10 +649,12 @@ class TPSTransformerLayer(MergeLayer):
     """
 
     def __init__(self, incoming, localization_network, downsample_factor=1,
-                 control_points=16, precompute_grid='auto', **kwargs):
+                 border_mode='nearest', control_points=16,
+                 precompute_grid='auto', **kwargs):
```

Comment:
Please add the new argument in the end, to keep compatibility with existing code that uses positional arguments.